src/server/Html.js

```diff
@@ -21,11 +21,17 @@ export default function Html({store, entries, StyleSheetServer, renderProps}) {
   // const initialState = `window.__INITIAL_STATE__ = ${JSON.stringify(store.getState())}`;
   // <script dangerouslySetInnerHTML={{__html: initialState}}/>
   const {html, css} = StyleSheetServer.renderStatic(() => {
-    return renderToString(
-      <Provider store={store}>
-        <RouterContext {...renderProps} />
-      </Provider>
-    );
+    try {
```

Comment:
ah, that's because i wrapped the whole client.js in an IIFE. we could put that in a then/catch clause & then in the client.js catch clause we could throw the error.